source/fab/parse/fortran.py

```diff
@@ -301,7 +301,11 @@ def _process_comment(self, analysed_file, obj):
             # TODO #13: once fparser supports reading the sentinels,
```

Comment:
An fParser issue number in the comment would help tracking this.

Reply:
Done.